control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -98,6 +99,7 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 sts_regional_endpoints = regional
 region = %s
 `
+	UserProvidedPullSecretNamespace = "kube-system"
```

Comment:
We'll likely need to change this, maybe to `pull-secret-user` or something like that.  We'll need interlock on docs for this.